src/main_chart.py

```diff
@@ -103,23 +103,42 @@ def get_main_chart_figure(
     color_map_liquidity = {"debt_token_supply": "#1f77b4"} # blue
     figure = plotly.graph_objs.Figure()
 
+    custom_columns = [  
```

Comment:
Move it to Constant.

If you have too large function it will be hard to read, so I would recommend to move out everything or create another function to make your code more readable = maintainable